app/models/user.rb

```diff
@@ -179,12 +178,41 @@ def resources
     user_permissions.map(&:resource)
   end
 
+  # #########################
+  # User Access (permissions)
+  #
+  def accessible_organizations(action)
```

Comment:
Could we use named arguments? Even though there's just one, I feel like these methods are definitely gonna be revisited and tweaked.

Reply:
I'll do this towards the end, since my later PRs and Vikram's PRs depend on this API, don't want that to break right now. Will stick a card for it.

Reply:
Haha that kinda validates why an interface that _doesn't_ require change across the codebase is nice 😂  Sounds good 👍